src/VisualStudio/Core/Def/Telemetry/VSTelemetryActivityLogger.cs

```diff
@@ -17,7 +17,6 @@ internal sealed class VSTelemetryActivityLogger : ForegroundThreadAffinitizedObj
         private static readonly HashSet<FunctionId> s_functionIds
             = new HashSet<FunctionId>()
             {
-                FunctionId.NavigateTo_Search,
```

Comment:
by the way, our framework is based on old VS telemetry API. they have all new API and I have work item for it.

https://github.com/dotnet/roslyn/issues/11895

no time to address it in RC though. anyway, once we move to new API, we probably merge 2 logger to one, but until then, we have 2 different one. one for perf, the other for information.